cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -181,15 +185,22 @@ func addCIFSInteractive(manager *hostFolderConfig.Manager, name string) error {
 		uncPath = util.ReadInputFromStdin("UNC path")
 	}
 
-	if len(uncPath) == 0 {
-		return errors.New("no remote path has been specified")
+	if uncPath == "" {
+		atexit.ExitWithMessage(1, noSource)
 	}
 
 	mountPoint := readInputForMountPoint(name)
 
 	username := util.ReadInputFromStdin("Username")
+	if username == "" {
```

Comment:
I guess you need to `TrimSpaces`

Reply:
Or did the function already do this? I forgot...

Reply:
It is not mentioned explicitly in the function, it using `fmt.Scanf` to read the input.. but white spaces are taken care of.

```
[minishift]$ minishift hostfolder add -i
Name:          
you need to specify a name
```
there are lot of spaces.. in the name